versions/build.ver

```diff
@@ -18,3 +18,4 @@ export sp_ver=2.4.0
 export w3emc_ver=2.9.2
 export wgrib2_ver=2.0.8
 export zlib_ver=1.2.11
+export bufr_ver=12.0.0
```

Comment:
@BinbinZhou-NOAA These are in alphabetical order. Can you please move `export bufr_ver=12.0.0` right after `export bacio_ver=2.4.1`?

CC @PerryShafran-NOAA @ShelleyMelchior-NOAA

Reply:
@BinbinZhou-NOAA Thank you for correcting run.ver for us! Quick question. Does export `bufr_ver=12.0.0` need to be added to build.ver? Is it needed to compile anything? If not, can you remove it from build.ver? Thanks!

CC @PerryShafran-NOAA @ShelleyMelchior-NOAA